instance/models/instance.py

```diff
@@ -455,10 +455,8 @@ class AnsibleInstanceMixin(models.Model):
     """
     An instance that relies on Ansible to deploy its services
     """
-    ansible_source_repo_url = models.URLField(max_length=256,
-                                              default='https://github.com/edx/configuration.git')
-
-    configuration_version = models.CharField(max_length=50, default='master')
+    ansible_source_repo_url = models.URLField(max_length=256, blank=True)
+    configuration_version = models.CharField(max_length=50, blank=True)
```

Comment:
Sorry, I realize that I'm pushing back on your whole PR, but I'd rather always have a value set for these fields; otherwise changing the default in the settings will change the value for all instances, which is what you meant to do here, but imho has unintended consequences - for example, it would also affects old instances, long after they have been deprovisioned, revising the history. Or if someone sets the configuration_version but not the repo_url, changing the default will break the instance provisioning if the commit isn't also on the fork. If we need to change the default, I'd rather explicitly update the fields, to ensure we only change values where it needs to.

Can we use the existing pattern for defaults set from the config, ie https://github.com/open-craft/opencraft/blob/545e0a2d43358f9ca0815e0aa9c19c62a5a144ce/instance/models/instance.py#L194-L197 ?